source/reference/s3-api-compatibility.rst

```diff
@@ -55,13 +63,11 @@ Multipart Uploads
 - :s3-api:`UploadPart <API_UploadPart.html>`
 - :s3-api:`UploadPartCopy <API_UploadPartCopy.html>`
 
-Unsupported API Object Endpoints
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. code-block:: text
+Differences from S3 APIs for Multipart Uploads
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   GetObjectAcl
-   PutObjectAcl
+- ``ListMultipartUploads`` requires the exact object name as a prefix
+- The ``AbortIncompleteMultipartUpload`` lifecycle action is not supported with ``PutBucketLifecycle``
```

Comment:
```suggestion
- The ``AbortIncompleteMultipartUpload`` lifecycle action is not supported with ``PutBucketLifecycle``
```

Adding ending punctuation as it's a complete sentence and can match the formatting of the last list on the page.